.github/workflows/auto-merge.yaml

```diff
@@ -8,6 +8,8 @@ env:
   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
   VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+  GITHUB_PRIVATE_KEY: ${{ secrets.VARGASJR_GITHUB_PRIVATE_KEY }}
+  GITHUB_REPOSITORY: ${{ github.repository }}
```

Comment:
We can remove both of these entirely. If the merge script has a reference to the repo name, I'm pretty sure there's a constant somewhere we could import